File: selenium/locators.rb
require <selenium:webdriver>
  
driver = Selenium::WebDriver.for :chrome
driver.find_element(:id, "username").send_keys("ishwarya")
driver.find_element(:id, "password").send_keys("ishh")
driver.find_element(:css, "button[type='submit']").click
</selenium:webdriver>

File: selenium/locators.js
const { Builder, By, Key } = require('selenium-webdriver');

async function login() {
  let driver = await new Builder().forBrowser('chrome').build();

  try {
    await driver.get("https://practicetestautomation.com/practice-test-login");

    await driver.findElement(By.id("username")).sendKeys("ISHWARYA");

    await driver.findElement(By.id("password")).sendKeys("mypassword");

    await driver.findElement(By.css("button[type='submit']")).click();

  } finally {
    await driver.sleep(2000);
    await driver.quit();
  }
}
//

